ci/L0_additional_outputs_vllm/test.sh

```diff
@@ -49,9 +49,8 @@ if [ "$SERVER_PID" == "0" ]; then
     exit 1
 fi
 set +e
-python3 additional_outputs_test.py > $CLIENT_LOG 2>&1
+python3 -m pytest -s -v additional_outputs_test.py
```

Comment:
nit: Please also dump the `--junitxml` report which should be auto-collected in our CI and visible on gitlab side, example: https://github.com/triton-inference-server/server/blob/462f31f6f22d327d88d480d5e01c7d1609f0a753/qa/L0_python_api/test.sh#L45

Reply:
Added: [pytest to dump additional outputs](https://github.com/triton-inference-server/vllm_backend/pull/70/commits/1773deafb683c463cde7dc6efc79f10d0eca381f)

Reply:
```
+ python3 -m pytest --junitxml=test_additional_outputs.xml -s -v additional_outputs_test.py
============================= test session starts ==============================
platform linux -- Python 3.10.12, pytest-8.1.1, pluggy-1.5.0 -- /usr/bin/python3
cachedir: .pytest_cache
rootdir: /opt/tritonserver/vllm_backend
configfile: pyproject.toml
plugins: anyio-4.6.2.post1
collecting ... collected 54 items
additional_outputs_test.py::TestAdditionalOutputs::test_additional_outputs[None-None-None-True] PASSED
...
additional_outputs_test.py::TestAdditionalOutputs::test_additional_outputs[False-False-False-False] PASSED
- generated xml file: /opt/tritonserver/vllm_backend/ci/L0_additional_outputs_vllm/test_additional_outputs.xml -
============================== 54 passed in 6.13s ==============================
```